conf.py

```diff
@@ -59,7 +59,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'AT-TPC Analysis'
+project = 'AT-TPC_Analysis'
```

Comment:
Oh, you're right. I think this was actually something stupid I did. If you change the "target name" in the LaTeX configuration [here](https://github.com/ATTPC/analysis-docs/blob/48b7eb87dfeed43b394db35cda30a8bf909988f0/conf.py#L217-L220) to something sensible that doesn't contain a space, it works. Right now, it just expands to `AT-TPC Analysis.tex` which is obviously a problem. I didn't think about that.